applications_tests/gls_vans_3d/beetstra_packed_bed.output

```diff
@@ -14,133 +14,133 @@ Void Fraction
 -------------------------------
 Volume-Averaged Fluid Dynamics
 -------------------------------
-Pressure drop: 40.47276122 Pa
-Total pressure drop: 40.47276154 Pa
-Global continuity equation error: 8.527914948e-13 s^-1
-Max local continuity error: 8.141667298e-08 s^-1
+Pressure drop: 39.62294819 Pa
```

Comment:
I was thinking, why not change this test to be a steady-state one now? The solver works very well in steady state nowadays.

Reply:
I don't know. Would you change all of them?